src/components/LoanCards/Buttons/MultipleAtcButton.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<div>
+		<button
+			class="tw-flex tw-items-center tw-pl-1.5"
+			style="font-size: 17px; font-weight: 611; border-radius: 16px; border: 1px solid #C4C4C4;"
+		>
+			${{ amount }}
+			<div
+				class="tw-ml-1.5"
+				style="background: #F4B539; font-weight: 621; border-radius: 0 14px 14px 0; padding: 12px 18px;"
```

Comment:
We should have a variable for this color now provided by kv-tokens. `tw-bg-marigold`

Reply:
Thank you @roger-in-kiva for having another look at these.

This one is under Primitives > Background Colors > Static. It's also helpful to toggle off dark mode. I'll make a note to stop loading the dark theme based on the browser settings b/c we don't officially support a full dark theme at this point and it makes it confusing to see weird colors :(

Reply:
Thanks to you for clarifying! Fixing right now.
Looks like I was working in a previous version, even asked me to updated node version...